app/test/obituary_test.rb

```diff
@@ -10,43 +10,42 @@ def app
 	end
 
   def test_it_returns_a_200
-  	get '/'
+  	get '/api/v1/'
```

Comment:
Thank you! this is pretty repetitive and was probably a pain in the butt, but definitely takes our microservice to the next level. Will definitely be showing this off in the README